### PR TITLE
[LWIMIQ] All Hands Busy

### PR DESCRIPTION
**Pull request by cybette**
Here's my first lwimiq entry. Sorry for my tardiness. 

@JPrause @jntullo @syncrou @imtayadeway be kind rewind. I mean review. :stuck_out_tongue_winking_eye:

### CODE REVIEW

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+This [PR][manageiq-providers-lenovo PR 76] from the [Lenovo XClarity provider plugin][manageiq-providers-lenovo] creates validation for credentials to the Lenovo provider, so that you don't have to turn your knuckles white with worry over invalid connections.
+
+## Jazz hands
+
+On the graphing front, [Graph refresh is turned on][manageiq-providers-amazon PR 290] for Amazon VMs with targeted refresh as the default. And what happens when you use GraphViz syntax for refresh graph logging? :open_hands: Tada - [pretty inventory graphs][manageiq PR 15814]! :open_hands:
+
+## Monkey tail
+
+Well, I selected some PRs I thought were interesting, in a totally unqualified way. If you could not make hands or tails out of it, or if you have other comments/suggestions for the project, please leave a feedback in our ongoing [ManageIQ Community Survey][miq survey], available for another week. As always, you're welcome to check out the progress we are making on [GitHub][miq github], and lend a hand!
```
Comment (imtayadeway) [inline site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: Think this possible needs to be "leave some feedback" or even "leave feedback"

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: "Last Week in ManageIQ: All Hands Busy"
+author: cybette
+date: 2017-09-19
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Last week, all engineering hands were busy in ManageIQ Engineering All-Hands meeting. A few stray hands still found the opportunity to submit some PRs, but there were no available fingers for typing up an entry for ["Last week in ManageIQ"][lwimiq tag]. So please allow me, [Carol][carol github], to paw in a few words in this edition.
```
Comment (syncrou) [inline site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: The `lwimiq tag` link returned a 404

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md (hunk 1 above)
Comment (syncrou) [inline site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: Also `miq github` returned a 404

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Green thumbs
+
+As mentioned during [last Sprint][Sprint 68], [Greenkeeper.io][greenkeeper] was added to [Service UI repo][manageiq-ui-service] to keep dependencies updated a lot faster.
+
+If you take a look at the [PRs merged][manageiq-ui-service PRs merged] in this repo last week, you will see many of them with green 'greenkeeper' labels, as various dependencies were updated to their latest versions.
+
+## Iron fists
+
+When it comes to paying technical debt, sometimes one has got to be ruthless. A bunch of constants [were][UI PR 2138] [placed][UI PR 2139] [on][UI PR 2141] [the][UI PR 2142] [chopping][UI PR 2145] [board][UI PR 2158], decreed to be removed from UiConstants. As those constants found new homes in appropriate modules, the UiConstants module was also [permanently deleted][UI PR 2165].
```
Comment (syncrou) [inline site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: Possibly 'board' could be switched to 'block'.

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md (hunk 2 above)
Comment (cybette) [reply site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: it's a relative URL, should be ok in production (it works in my dev env)

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md (hunk 3 above)
Comment (cybette) [reply site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: yep! I was probably hungry...

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md (hunk 1 above)
Comment (cybette) [reply site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: also a relative URL. Thanks for checking the links!!

**Comment by cybette** [comment]
Thanks @syncrou @imtayadeway for review! @Fryguy please check and merge!

File: site/_posts/2017-09-19-lwimiq-all-hands-busy.md (hunk 1 above)
Comment (cybette) [reply site/_posts/2017-09-19-lwimiq-all-hands-busy.md]: @JPrause yeah these younglings :stuck_out_tongue:

**Comment by JPrause** [comment]
LGTM 👍  @Fryguy can you merge since we're a few days late posting this week's blog entry.

**Pull request merged**